core/aggregator/api/api_domain.go

```diff
@@ -50,7 +50,7 @@ func CreateDomain(w http.ResponseWriter, r *http.Request) {
 		var domain *models.Domain
```

Comment:
I think you can squash the commit "update product" with previous. But take care of another comment for the separation I make.